hack/build-antrea-linux-all.sh

```diff
@@ -169,22 +163,16 @@ if [ "$DISTRO" == "ubuntu" ]; then
     if $COVERAGE; then
         make build-controller-ubuntu-coverage
         make build-agent-ubuntu-coverage
-        if $UNIFIED; then
-            make build-ubuntu-coverage
-        fi
+        make build-ubuntu-coverage
     else
         make build-controller-ubuntu
         make build-agent-ubuntu
```

Comment:
instead of calling `make build-agent-ubuntu` and `make build-controller-ubuntu` can we just call `make`

Reply:
There are 3 cases, I keep the two calls to make code symmetric

Reply:
okay